### ISSUE TITLE
Interpretation of output

### ISSUE THREAD

**Issue by piseabhijeet** [Sibozhu/grad-cam-formula-student#2]
Hi @Sibozhu 

I am using a custom trained Tiny YOLOv3 model with 5 classes for GradCAM.

I noticed the following:
![image](https://user-images.githubusercontent.com/57705684/106568374-3758e700-6559-11eb-8fcb-2ddb99253972.png)

The output of my model is 
13 x 13 x 3 x (5+5) +
26 x 26 x 3 x (5+5) thus giving the output shape in your code as:
2535 x 10

However the variable 'probs[0]' and 'idx[0]' each contain 10 elements with different bboxes than the original Darknet
I am getting 4 bboxes with confidence as 1 and in this code i am getting 2 boxes with confidence 1. 

Are 'probs[0]' and 'idx[0]' the right inputs for gradcam or am i missing something?

**Comment by adityag6994** [comment]
@piseabhijeet did you figure out the reason, has it something to do with that originally YOLOv3 uses sigmoid for class loss, and not soft-max ? 
I'm also confused from this line here : [link](https://github.com/Sibozhu/grad-cam-formula-student/blob/0616d17eb7435ad1f0960a3b6b4feaab78ea7933/src/utils/grad_cam.py#L42) , not able to relate it to original paper. Please clarify if you know the reason. 

In simpler terms, I think, it's the loss calculation I am confused about.

Thanks.
